data/StevenBlack/hosts

```diff
@@ -3052,4 +3052,7 @@
 0.0.0.0 zipservice.adalliance.io
 
 # Added February 28, 2021
-0.0.0.0 github.com.cnpmjs.org
\ No newline at end of file
+0.0.0.0 github.com.cnpmjs.org
+
+# Added March 15, 2021
+0.0.0.0 facilitategovernor.com
```

Comment:
@FadeMind you can add those as well,if you want:
`0.0.0.0 www.facilitategovernor.com`
`0.0.0.0 mvideo.facilitategovernor.com`
`0.0.0.0 ns1.facilitategovernor.com`